pkg/ui/workspaces/cluster-ui/src/insights/workloadInsights/util/insightsColumns.tsx

```diff
@@ -111,6 +121,18 @@ export const insightsTableTitles: InsightsTableTitleType = {
       "username",
     );
   },
+  schemaName: (execType: InsightExecEnum) => {
+    return makeToolTip(<p>The name of the contended schema.</p>, "schemaName");
+  },
+  databaseName: (execType: InsightExecEnum) => {
+    return makeToolTip(
+      <p>The name of the contended database.</p>,
+      "databaseName",
+    );
+  },
+  tableName: (execType: InsightExecEnum) => {
+    return makeToolTip(<p>The name of the contended table.</p>, "tableName");
+  },
   applicationName: (execType: InsightExecEnum) => {
```

Comment:
@j82w @kevin-v-ngo 
>  is there a reason it does not display the index?

The `WaitTimeInsightsPanel` displays a contended secondary index if there is one:

https://github.com/cockroachdb/cockroach/blob/master/pkg/ui/workspaces/cluster-ui/src/detailsPanels/waitTimeInsightsPanel.tsx#L114

If there isn't one, then it won't display it. We probably want an `indexName` column in the blocking transactions table too, but that value might be blank.

Reply:
Is there a reason why we wouldn't have a contended index? And even key/row span? I thought contention would always be at the key level?

Reply:
@kevin-v-ngo 

> Is there a reason why we wouldn't have a contended index?

I think that depends on the query. 

> And even key/row span? I thought contention would always be at the key level?

We remove the contended key from the txn insight details page. 

Discussion thread:
https://cockroachlabs.slack.com/archives/G01Q9D01NTU/p1662676417756419?thread_ts=1662662201.540079&cid=G01Q9D01NTU

PR:
https://github.com/cockroachdb/cockroach/pull/87645